source/material_model/interface.cc

```diff
@@ -236,7 +236,7 @@ namespace aspect
     }
 
 
-
+    DEAL_II_DISABLE_EXTRA_DIAGNOSTICS
```

Comment:
comments never hurt, but I primarily wanted to understand it from the perspective of a reviewer :-)